plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,521 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024, Cisco Systems
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import time
+import pandas as pd
+from dnacentersdk import api
+from ansible_collections.cisco.dnac.plugins.module_utils.dnac import (
+    DnacBase
+)
+from ansible_collections.cisco.dnac.plugins.module_utils.pydentic_validation import *
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+__author__ = ("A Mohamed Rafeek, Natarajan")
+
+DOCUMENTATION = r"""
+---
+module: accesspoint_workflow_manager
+short_description: accesspoint_workflow_manager used to automate bulk AP configuration changes.
+description:
+- We can change the AP display name, AP name or Other Param based on the input.yml file
+- Using by this package we can filter specific device details like family = Switches and Hubs
+- We can compare input details with current AP configuration.
+- Desired configuration will be updated to the needed APs Only
+- Also able to reboot the Accesspoint if needed.
+version_added: '6.7.0'
+extends_documentation_fragment:
+  - accesspoint_workflow_manager
+author: A Mohamed Rafeek (@mohamedrafeek)
+        Natarajan (@natarajan)
+
+options:
+    config_verify:
+        description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
+    type: bool
+    default: False
+    state:
+        description: The state of Cisco Catalyst Center after module completion.
+    type: str
+    choices: [ merged ]
+    default: merged
+
+config:
```

Comment:
This should be defined under options only and define the each parameter along with it's type, description, required or not.

Reply:
moved to under options